jbpm-human-task/jbpm-human-task-audit/src/test/java/org/jbpm/services/task/audit/service/TaskAuditBaseTest.java

```diff
@@ -1328,6 +1327,33 @@ public void testUpdateTaskContentEvents() {
         Assertions.assertThat(taskEvents.get(5).getType()).isEqualTo(TaskEventType.COMPLETED);
         Assertions.assertThat(taskEvents.get(5).getUserId()).isEqualTo("Darth Vader");
     }
+    
+    @Test
+    public void testForwardTaskWithMsgEvents() {
+        Task task = new TaskFluent().setName("This is my task name")
+                .addPotentialGroup("Knights Templer")
+                .setAdminUser("Administrator")
```

Comment:
@mcivantos-tribalyte you should also explicitly add _Administrator_ as a potential owner, if you want to forward() this task. This should fix the permission exception.

Reply:
to follow up I thought it not needed according to what is written in WS-HT spec, Administrator must be a potential owner if he wants to be able to forward